src/notfication/notify.py

```diff
@@ -0,0 +1,28 @@
+from email.message import EmailMessage
+import smtplib
+import os
+import ssl
+# from dotenv import load_dotenv
+# load_dotenv()
+
+class Notify:
```

Comment:
Let's rename this something like `EmailNotification` and please change the file name to `email.py` as well.
As to why this change is needed, you can have a look at this [issue](https://github.com/BajajGirik/deal-finder/issues/10).

Reply:
changing the name to email.py will lead to some dependency conflicts

Reply:
Strange. I fixed and tested it out now.